client/my-sites/stats/stats-list/action-spam.jsx

```diff
@@ -37,47 +37,50 @@ class StatsActionSpam extends React.Component {
 			this.props.afterChange( ! this.state.spammed );
 		}
 
-		wpcomSite = wpcom.site( this.props.data.siteID );
+		const wpcomSite = wpcom.site( this.props.data.siteID );
 		wpcomSite[ spamType ].call( wpcomSite, this.props.data.domain, function() {} );
 		analytics.ga.recordEvent( 'Stats', gaEvent + ' in ' + this.props.moduleName + ' List' );
 	};
 
 	render() {
-		let label = this.state.spammed
-				? this.props.translate( 'Not Spam' )
-				: this.props.translate( 'Spam', {
-						context: 'Stats: Action to mark an item as spam',
-						comment: 'Default label (changes into "Not Spam").',
-				  } ),
-			title = this.state.spammed
-				? this.props.translate( 'Not Spam', {
-						textOnly: true,
-						context: 'Stats: Action to undo marking an item as spam',
-						comment:
-							'Secondary label (default label is "Spam"). Recommended to use a very short label.',
-				  } )
-				: this.props.translate( 'Spam', {
-						textOnly: true,
-						context: 'Stats: Action to mark an item as spam',
-						comment: 'Default label (changes into "Not Spam").',
-				  } ),
-			wrapperClass = classNames( 'module-content-list-item-action-wrapper', {
-				spam: ! this.state.spammed,
-				unspam: this.state.spammed,
-			} );
+		const label = this.state.spammed
+			? this.props.translate( 'Mark as Not Spam' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **45** times:
`translate( 'Mark as Not Spam' )` *ES Score: 12*


:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?